packages/docsearch-react/package.json

```diff
@@ -20,6 +20,11 @@
     "button.js",
     "modal.js"
   ],
+  "exports": {
+    ".": "./dist/esm/index.js",
+    "./button": "./dist/esm/DocSearchButton.js",
+    "./modal": "./dist/esm/DocSearchModal.js"
+  },
```

Comment:
For Docusaurus, we would also need entry points to consume these without pulling the full code

```
import {version, useDocSearchKeyboardEvents} from '@docsearch/react';

```